Makefile

```diff
@@ -67,6 +68,9 @@ dockertestshell:
 dockerdocs:
 	./docker/run_build_docs.sh
 
+dockerupdatedata:
+	./docker/run_update_data.sh
+
```

Comment:
I created this make rule in this PR because I tweaked the script to use the crontabber container, but it's not actually related to the crontabber container change.